package.json

```
@@ -1797,7 +1797,8 @@
                 "docker.imageBuildContextPath": {
                     "type": "string",
                     "default": "",
-                    "description": "%vscode-docker.config.docker.imageBuildContextPath%"
+                    "description": "%vscode-docker.config.docker.imageBuildContextPath%",
+                    "scope": "machine"
```

```suggestion
                    "scope": "machine-overridable"
```
Don't we want this to be configurable per workspace/folder?

---

Sure, that's fair enough. I don't have specific scenarios in mind to be pro/against so we can do it this way and see the feedback.